e2e/preferences.e2e.spec.ts

```diff
@@ -0,0 +1,142 @@
+import os from 'os';
+import path from 'path';
+import { ElectronApplication, BrowserContext, _electron, Page } from 'playwright';
+import { test, expect } from '@playwright/test';
+import { createDefaultSettings, packageLogs, reportAsset } from './utils/TestUtils';
+import { NavPage } from './pages/nav-page';
+import { PreferencesPage } from './pages/preferences';
+
+let page: Page;
+
+/**
+ * Using test.describe.serial make the test execute step by step, as described on each `test()` order
+ * Playwright executes test in parallel by default and it will not work for our app backend loading process.
+ * */
+test.describe.serial('Main App Test', () => {
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+  let preferencesWindow: Page;
+
+  test.beforeAll(async() => {
+    createDefaultSettings();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        // See src/utils/commandLine.ts before changing the next item as the final option.
+        '--disable-dev-shm-usage',
+        '--no-modal-dialogs',
+      ],
+      env: {
+        ...process.env,
+        RD_LOGS_DIR: reportAsset(__filename, 'log'),
```

Comment:
I don't believe any of this depends on the backend; as such, please set `RD_MOCK_BACKEND` to `1` here to get a faster test.

Reply:
Doing so produces a failure when fetching kubernetes versions 

<img width="880" alt="Screen Shot 2022-07-18 at 8 09 51 PM" src="https://user-images.githubusercontent.com/835961/179656444-f944e1ae-0817-4a30-a2ac-449f6fba1d82.png">